docker/pandoc-lambda/function/app.py

```diff
@@ -52,7 +171,19 @@ def handler(event, context):
                         sig_string = f"unknown signal {-e.returncode}"
                 else:
                     sig_string = f"non-zero exit status {e.returncode}"
-                raise Exception(f"Pandoc command exited with {sig_string}")
+                ss = "Pandoc command exited with " + str(sig_string)
+                print(ss)
+                pandoc_out.seek(0,0)
+                print(response.stderr)
+                raise Exception(ss)
             if not os.path.getsize(pandoc_out.name) > 0:
                 raise Exception(f"Pandoc produced no output.")
+
+            if options.get('word_footnotes', False):
+                doc = Document(pandoc_out)
+                promote_case_footnotes(doc)
+                output = io.BytesIO()
```

Comment:
I've been interested in whether we want to be working with temp files or with bytes IO objects during this whole project. If we've made it this far in the process and not run out of RAM, then surely we have enough to use a bytes IO here. We get charged the same, regardless of whether we actually USE all the RAM we have allocated, so it's not a win in that way to write to disk. 

Would you be willing to comment if you picked bytes IO for speed, convenience, etc., or if you use it as your go to (over, say, a spooling temp file), or what have you?

Reply:
My general thought was that pandoc would have just freed up all the RAM it was using, so there'd be plenty of space free for the taking. I suspect it's marginally faster, though probably not enough for anyone to notice. I think this was probably more of a desire to not open up a second temp file, or go through the trouble of erasing the first to overwrite it.

Reply:
👍 thx!